ReactWindows/ReactNative/UIManager/BaseViewManager.cs

```diff
@@ -188,6 +190,73 @@ public void SetTooltip(TFrameworkElement view, string tooltip)
             ToolTipService.SetToolTip(view, tooltip);
         }
 
+        /// <summary>
+        /// Sets the shadow color of the <typeparamref name="TFrameworkElement"/>.
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="color">The shadow color.</param>
+        [ReactProp("shadowColor", CustomType = "Color")]
+        public void SetShadowColor(TFrameworkElement view, uint? color)
+        {
+            if (color.HasValue)
+            {
+                var hostVisual = (SpriteVisual)ElementCompositionPreview.GetElementChildVisual(GetOrCreateShadowHost(view));
+                if (hostVisual.Shadow is DropShadow shadow)
+                {
+                    shadow.Color = ColorHelpers.Parse(color.Value);
+                }
+            }
+            else
+            {
+                RemoveShadow(view);
+            }
```

Comment:
Looks like this could be a brittle approach. Here's an issue repro:
1) Set `shadowColor` to non-null (e.g., 'black')
2) Set `shadowOpacity` to 0.5
3) Set `shadowColor` to null
4) Set `shadowColor` to non-null (e.g., 'black')

In this case, when I set `shadowColor` to null, I lose the state for `shadowOpacity`. When I then set `shadowColor` back to non-null, it will be as if I never set `shadowOpacity`.

Reply:
Fair enough. I'll take a look at tracking the properties more thoroughly!

I didn't originally since no other properties seemed to be tracked this way. Eg, setting `borderColor` before `borderWidth` results in the color being ignored/forgotten.

Reply:
Question:
I have a robust property tracking system working now, but I'm not sure what the desired/expected behavior is.

Let's say I have a `View` and I change its style a bit based  on some event. Say it goes through the following sequence of styles:

```
{ shadowColor 'black', shadowRadius: 12 }
{ shadowRadius: 12 }
{}
{ shadowColor: 'green' }
```

What is the expected display output for each step?

Reply:
Possibly relevant to the discussion is that i can't figure out a way to distinguish if a property is explicitly set to null or if it was just removed from the style. So, any time above where the style changes to exclude a property, it is currently equivalent to explicitly setting it to null.

In other words, the above sequence of style states is (as far as I can determine) impossible to distinguish from this one:

```
{ shadowColor 'black', shadowRadius: 12 }
{ shadowColor: null, shadowRadius: 12 }
{ shadowColor: null, shadowRadius: null }
{ shadowColor: 'green', shadowRadius: null }
```

Though I would argue the first version is the more likely way to see these properties being set.

Reply:
Pushed a commit that treats this sequence the--to me--least surprising way, from the perspective of a user.

```
 // A black shadow with radius 12 is shown. The other props are defaulted
// (as per DropShadow's defaults for them)
{ shadowColor 'black', shadowRadius: 12 }

// A shadow with radius 12 is shown. All the other props are defaulted as above,
// including color.
{ shadowRadius: 12 }

// The shadow is removed
{}

// A green shadow is shown. All the other props are defaulted again.
{ shadowColor: 'green' }
```